get

```diff
@@ -24,7 +24,11 @@ verifyGoInstallation() {
 		fail "$PROJECT_NAME needs go. Please install it first."
 	fi
 	if [ -z "$GOPATH" ]; then
+	    # Verify if default GOPATH of Go >=1.8 is used
+	    GOPATH="${HOME}/go"
```

Comment:
You can execute the command `go env GOPATH` to get the current `GOPATH`. What about using that instead?